### ISSUE TITLE
Has someone implemented boundaries yet?

### ISSUE THREAD

**Issue by MIMNSI** [mkkellogg/GaussianSplats3D#303]
It would be great if we can limit the area where camera can be panned. Example: For interior scenes where the camera passes through walls at the moment.

**Comment by newguy123-creator** [comment]
we do that within a three js scene and a custom camera controller.
You can get fancy if you like and also add physics and invisible walls as collider objects.

**Comment by MIMNSI** [comment]
Right, I've been working with Navmeshes currently within a threejs scene. Thinking on the same lines as you.

**Comment by ssswhite007** [comment]
Hello, @MIMNSI 
I am also trying to implement boundaries as you.
Hope to ask that you implement boundaries. :)
Can you help me?

**Comment by seppestaes** [comment]
Hi,
Just a couple of ideas:
- Use the cameras.json path (COLMAP) as camera boundaries
- Raytracing to find intersecting splats and remove from scene (texture mod)
